### PR TITLE
8 collapse samples

### PR DESCRIPTION
**Pull request by wasade**
depends on #166, #167, #168

### CODE REVIEW

File: ipynb/primary-processing/8-collapse_samples.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+>>> ag_100nt_1k_oral_diet_biom      = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-diet-biom'])
+>>> ag_100nt_1k_oral_age_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-age-biom'])
+>>> ag_100nt_1k_oral_flossing_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-flossing-biom'])
+...
+>>> ag_100nt_1k_skin_sex_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-sex-biom'])
+>>> ag_100nt_1k_skin_cosmetics_biom = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-cosmetics-biom'])
+>>> ag_100nt_1k_skin_age_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-age-biom'])
+>>> ag_100nt_1k_skin_hand_biom      = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-hand-biom'])
+...
+>>> ignored = 'foo'
```
Comment (squirrelo) [inline ipynb/primary-processing/8-collapse_samples.md]: Is this a placeholder for future use? It's only set here

Comment (wasade) [inline ipynb/primary-processing/8-collapse_samples.md]: Can add a comment if you'd like, but there was a file path that was necessary for the call but not used after the fact

Comment (squirrelo) [inline ipynb/primary-processing/8-collapse_samples.md]: Ah, yeah a comment would be good.

**Comment by squirrelo** [comment]
Just one comment.

**Comment by wasade** [comment]
merge?

**Comment by josenavas** [comment]
@coveralls 

giphy

**Pull request merged**